### PR TITLE
Winterfell crate

### PR DESCRIPTION
**Pull request by irakliyk**
This PR will add `winterfell` create which re-exports prover and verifier crates. The primary purpose of this is to simplify crate imports in external projects.

- [x] Crete `winterfell` create
- [x] Update `examples` create
- [x] Update documentation

### CODE REVIEW

File: examples/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [package]
-name = "winterfell"
+name = "examples"
```
Comment (kchalkias) [inline examples/Cargo.toml:2]: I would prefer a better name here, i.e., applications or something similar (I prefer gadgets, but unfortunately they are not necessarily reusable, so let's pick something else).

File: winterfell/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+//!    result
+//! }
+//! ```
+//!
+//! This computation starts with an element in a finite field and then, for the specified number
+//! of steps, cubes the element and adds value `42` to it.
+//!
+//! Suppose, we run this computation for a million steps and get some result. Using STARKs we can
+//! prove that we did the work correctly without requiring any verifying party to re-execute the
+//! computation. Here is how to do it.
```
Comment (kchalkias) [inline winterfell/src/lib.rs:81]: replace . with :

**Review by kchalkias — APPROVED** [review]
Approved, minor comments (mainly to rename "examples" in another PR)

File: winterfell/src/lib.rs (hunk 2 above)
Comment (irakliyk) [reply winterfell/src/lib.rs]: Thanks! Fixed!

File: examples/Cargo.toml (hunk 1 above)
Comment (irakliyk) [reply examples/Cargo.toml:2]: I thing `gadgets` makes sense to me. Will rename in another PR.

**Pull request merged**